### PR TITLE
testing one two

### PR DESCRIPTION
**Pull request by ming1in**
adsfasdfdsf

### CODE REVIEW

**Review by aalobai1 — APPROVED** [review]

**Comment by ming1in** [comment]
sdfdsfdfsdf

**Comment by aalobai1** [comment]
Disney

**Review by aalobai1 — COMMENTED** [review]
Hd

**Review by aalobai1 — APPROVED** [review]
Approve

File: app.ts [hunk 1]
```diff
@@ -122,6 +122,10 @@ const onSubmitPullRequestReview = async (
   const { review, pull_request } = payload;
 
   console.log("onSubmitPullRequestReview(): " + review.state);
+
+  if (review.state === "APPROVED") {
+    console.log("THIS IS A APPROVED EVENT")
```
Comment (coreyja) [inline app.ts:127]: Lets take this to Slack!

File: app.ts [hunk 2]
```diff
@@ -122,6 +122,10 @@ const onSubmitPullRequestReview = async (
   const { review, pull_request } = payload;
 
   console.log("onSubmitPullRequestReview(): " + review.state);
+
+  if (review.state === "APPROVED") {
```
Comment (coreyja) [inline app.ts:126]: Hey @ming, lets make a thread here! Add a comment or two!

Comment (ming1in) [reply app.ts:126]: hey this code could be better

Comment (ming1in) [reply app.ts:126]: i think this breaks HIPAA

Comment (coreyja) [reply app.ts:126]: It's looking pretty good for 11am! 😆

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (ming1in) [reply app.ts:126]: nit: i think this needs to be refactored

Comment (coreyja) [reply app.ts:126]: lets take this to slack

Comment (coreyja) [reply app.ts:126]: take this to slack!!!

Comment (coreyja) [reply app.ts:126]: take this to slack

Comment (coreyja) [reply app.ts:126]: take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

Comment (coreyja) [reply app.ts:126]: Lets take this to slack

File: app.ts [hunk 3]
```diff
@@ -122,6 +122,10 @@ const onSubmitPullRequestReview = async (
   const { review, pull_request } = payload;
 
   console.log("onSubmitPullRequestReview(): " + review.state);
```
Comment (coreyja) [inline app.ts:124]: Lets start a new group

Comment (coreyja) [reply app.ts:124]: And now lets take it to slack?

Comment (coreyja) [reply app.ts:124]: take this to slack

Comment (coreyja) [reply app.ts:124]:
```
what happens with code blocks?
```

Comment (coreyja) [reply app.ts:124]: lets that this to slack

Comment (coreyja) [reply app.ts:124]: lets take this to slack

Comment (coreyja) [reply app.ts:124]: lets take this to slack

**Pull request closed without merging**